src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -185,11 +185,20 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
         protected override async Task<bool> HandleUnauthorizedAsync(ChallengeContext context)
         {
-            var authResult = await HandleAuthenticateOnceAsync();
+            Exception failure = null;
```

Comment:
Do we really ever want HandleAuthenticateOnceAsync to throw?  Should we just make this failsafe and push this try/catch into HandleAuthenticateOnce, to always return at least a non null authResult with a Failure or Success.

Its pretty gross if this is what needs to be done everywhere we call HandleAuthenticateOnceAsync